### PR TITLE
Hunt1

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: tests/slow/src/test/scala/org/scalafmt/internal/SyntaxAnalysis.scala [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    * @return The aggregate sum of all analysis results.
+    */
+  def run[T](corpus: GenIterable[CorpusFile])(
+      f: CorpusFile => List[T]
+  ): mutable.Buffer[(CorpusFile, T)] = Phase.run("syntax analysis") {
+    val results = new CopyOnWriteArrayList[(CorpusFile, T)]
+    val counter = new AtomicInteger()
+    val errors  = new AtomicInteger()
+    def analyze(file: CorpusFile): Unit = {
+      try {
```
Comment (MasseGuillaume) [inline tests/slow/src/test/scala/org/scalafmt/internal/SyntaxAnalysis.scala]: this is to remove a println screwing the progress bar

File: tests/slow/src/test/scala/org/scalafmt/internal/IdempotencyPropertyTest.scala [hunk 2]
```diff
@@ -88,6 +94,7 @@ object IdempotencyPropertyTest extends BaseScalaPrinterTest {
           } else {
             val diff = getDiff(file.jFile.getAbsolutePath, treeNorm, tree2Norm)
             if (diff.nonEmpty) {
+              println("")
```
Comment (MasseGuillaume) [inline tests/slow/src/test/scala/org/scalafmt/internal/IdempotencyPropertyTest.scala:90]: show the diff on the next line after the progress bar

File: tests/slow/src/test/scala/org/scalafmt/internal/SyntaxAnalysis.scala [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import scala.collection.GenIterable
+import scala.collection.JavaConverters._
+import scala.collection.mutable
+import scala.meta._
+import scala.util.control.NonFatal
+
+import java.util.concurrent.CopyOnWriteArrayList
+import java.util.concurrent.atomic.AtomicInteger
+
+object SyntaxAnalysis {
```
Comment (olafurpg) [inline tests/slow/src/test/scala/org/scalafmt/internal/SyntaxAnalysis.scala:14]: Can we add `// + scalameta deviation` `// - scalameta deviation` comments to distinguish what has changed?

File: tests/unit/src/test/scala/org/scalafmt/tests/TermMatchSuite.scala [hunk 4]
```diff
@@ -10,5 +10,4 @@ object TermMatchSuite extends BaseScalaPrinterTest {
   check("a match { case 1 => }")
   check("(a: @switch) match { case _ => b }")
   check("(if (a) b else c) match { case 1 => }")
-  check("a match {case _ => { f op { b } } }")
```
Comment (olafurpg) [inline tests/unit/src/test/scala/org/scalafmt/tests/TermMatchSuite.scala:13]: Why is this test removed? If the output has changed we can update the test. the test is there only for ast changes

File: format/src/main/scala/org/scalafmt/internal/TreePrinter.scala [hunk 5]
```diff
@@ -472,7 +472,8 @@ object TreePrinter {
         dDef(t.mods, `def`, `this`, Nil, t.paramss, None, dbody)
       case m: Mod =>
         m match {
-          case t: Mod.Annot => `@` + print(t.init)
+          case t: Mod.Annot =>
+            `@` + SimpleTyp.wrap(t.init.tpe) + dArgss(t.init.argss)
```
Comment (olafurpg) [inline format/src/main/scala/org/scalafmt/internal/TreePrinter.scala:476]: Can we keep this on a single like like the rest of the `Mod` cases

**Review by olafurpg — CHANGES_REQUESTED** [review]
Woah, close to 100%

File: tests/slow/src/test/scala/org/scalafmt/internal/SyntaxAnalysis.scala (hunk 3 above)
Comment (MasseGuillaume) [reply tests/slow/src/test/scala/org/scalafmt/internal/SyntaxAnalysis.scala:14]: I removed a println

File: tests/unit/src/test/scala/org/scalafmt/tests/TermMatchSuite.scala (hunk 4 above)
Comment (MasseGuillaume) [reply tests/unit/src/test/scala/org/scalafmt/tests/TermMatchSuite.scala:13]: I think the ci is disabled, this should not have passed.

File: format/src/main/scala/org/scalafmt/internal/TreePrinter.scala (hunk 5 above)
Comment (MasseGuillaume) [reply format/src/main/scala/org/scalafmt/internal/TreePrinter.scala:476]: `scalafmt` says nope

**Review by olafurpg — APPROVED** [review]

**Comment by MasseGuillaume** [comment]
ci is still disabled

**Pull request merged**